com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -120,45 +121,22 @@ void DelayedSpawnAction(Stream continuationStream)
                             NetworkManager.Singleton.SpawnManager.ClientCollectSoftSyncSceneObjectSweep(null);
                         }
 
-                        uint objectCount = continuationReader.ReadUInt32Packed();
+                        var objectCount = continuationReader.ReadUInt32Packed();
                         for (int i = 0; i < objectCount; i++)
                         {
-                            bool isPlayerObject = continuationReader.ReadBool();
-                            ulong networkId = continuationReader.ReadUInt64Packed();
-                            ulong ownerId = continuationReader.ReadUInt64Packed();
-                            bool hasParent = continuationReader.ReadBool();
+                            var isPlayerObject = continuationReader.ReadBool();
+                            var networkId = continuationReader.ReadUInt64Packed();
+                            var ownerId = continuationReader.ReadUInt64Packed();
```

Comment:
are these more accurate?

```suggestion
                            var networkObjectId = continuationReader.ReadUInt64Packed();
                            var ownerClientId = continuationReader.ReadUInt64Packed();
```

Reply:
Yeah, I noticed the names but decided to leave them out for this PR.  This area will get reworked.  For now, I am going to keep all of the names the same as it s used in several places.